react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,53 @@
+.sidebar-off {
+  height: 100%;
+  width: 0px;
+  transition: 0.5s;
+  display: inline-block;
+  background-color:#ffffff;
+  padding-top: 1rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
+
+.sidebar-on {
+  height: 100%;
+  width: 250px;
+  transition: width 0.5s;
```

Comment:
Transitioning the width is usually really stuttery and makes the content of the element squish. Try leaving the width as one value and transitioning the `transform: translateX()` instead.